diagnostics/collect-wsl-logs.ps1

```diff
@@ -130,6 +130,70 @@ if ($Dump)
     }
 }
 
+# Collect networking state relevant for WSL
+$stdOutLog = "$folder/wsl_network_stdout.log"
+$stdErrLog = "$folder/wsl_network_stderr.log"
+
+# Using a try/catch as some of the commands below do not exist on some OS versions
```

Comment:
Would it be better to wrap individual operations in try / catch so collection doesn't stop at first error?

Reply:
good point, will update